appstore/model.go

```diff
@@ -48,14 +57,23 @@ type (
 		ProductID                 string `json:"product_id"`
 		TransactionID             string `json:"transaction_id"`
 		OriginalTransactionID     string `json:"original_transaction_id"`
-		IsTrialPeriod             string `json:"is_trial_period"`
 		AppItemID                 string `json:"app_item_id"`
 		VersionExternalIdentifier string `json:"version_external_identifier"`
 		WebOrderLineItemID        string `json:"web_order_line_item_id"`
+
+		IsTrialPeriod                  string `json:"is_trial_period"`
+		SubscriptionExpirationIntent   string `json:"expiration_intent"`
```

Comment:
The Apple's document is ambiguous as you say :-(
I think it is better to remove the fields which are not existing in the raw response.
Otherwise, someone uses it by mistake.

Could you remove them? (with ``app_item_id`` and ``version_external_identifier`` in ``InApp``)